src/Summoner/Config.hs

```diff
@@ -151,14 +154,15 @@ configT = Config
     <*> lastT (Toml.table preludeT) "prelude" .= cPrelude
     <*> textArr         "extensions"  .= cExtensions
     <*> textArr         "warnings"    .= cWarnings
+    <*> lastT sourceT   "stylish"     .= cStylish
```

Comment:
Okay, I see problem in this parser. Looks like `maybeT` parser that I've implemented doesn't work 😞 It has the following type:

```haskell
maybeT :: forall a . (Key -> BiToml a) -> Key -> BiToml (Maybe a)
```

The problem is in the fact that if either `stylish.url` or `stylish.file` absent, `maybeT` checks whether exception is `KeyNotFound "stylish"` or `TableNotFound "stylish"`. But the actual exception is none of this! So looks like `maybeT` combinator is completely broken.

I propose to remove `maybeT` completely from `tomland` in next major release. Since `BiToml` has an `Alternative` instance, you can use [`optional`](https://hackage.haskell.org/package/base-4.11.1.0/docs/Control-Applicative.html#v:optional) to achieve semantic of `maybeT`. 

Currently we have:

```haskell
    <*> lastT sourceT   "stylish"     .= cStylish

lastT :: (Key -> BiToml a) -> Key -> BiToml (Last a)
lastT = Toml.wrapper . Toml.maybeT
```

The fix is to write like this:

```haskell
    <*> wrapLastT (maybeSourceT "stylish") .= cStylish

wrapLastT :: BiToml (Maybe a) -> BiToml (Last a)
wrapLastT = Toml.dimap getLast Last

maybeSourceT :: Key -> BiToml (Maybe Source)
maybeSourceT key = optional (sourceT key)
```